### ISSUE TITLE
"umfpack.h" missing from version 1.1.9 tarball?

### ISSUE THREAD

**Issue by whoburg** [cvxopt/cvxopt#77]
We're suddenly having trouble building cvxopt on Linux (Ubuntu), and the problem seems to be related to `umfpack.h`.

```
  building 'umfpack' extension
  x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fno-strict-aliasing -D_FORTIFY_SOURCE=2 -g -fstack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security -fPIC -I/usr/include -I/usr/include/python2.7 -c src/C/umfpack.c -o build/temp.linux-x86_64-2.7/src/C/umfpack.o
  src/C/umfpack.c:23:21: fatal error: umfpack.h: No such file or directory
   #include "umfpack.h"
                       ^
  compilation terminated.
  error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
```

**Comment by galbramc** [comment]
I work with @whoburg. Looks like the problem is actually here in setup.py

```
# Guess SUITESPARSE_LIB_DIR and SUITESPARSE_INC_DIR
lnx_dist = platform.linux_distribution()
if lnx_dist[0] == "Ubuntu" and lnx_dist[1] >= "16":
    # Ubuntu 16.04 or later
    SUITESPARSE_LIB_DIR = "/usr/lib/x86_64-linux-gnu"
    SUITESPARSE_INC_DIR = "/usr/include/suitesparse"
```

We are using Ubuntu 14.04. However, this is actually valid going back as far as 12.04 (that's the oldest version we still have running). So changing to 12.04 or later would be just fine.

**Comment by martinandersen** [comment]
We do not include the SuiteSparse source anymore. You will either need a shared suitesparse llibrary (available via the package manager in Ubuntu 16.04, but no in 14.04) or to download the suitesparse source separately and set `CVXOPT_SUITESPARSE_SRC_DIR`. Please refer to the updated installation instructions for details.

**Comment by galbramc** [comment]
@martinandersen. There is a suitesparse-dev package available in 14.04, but there is a problem with the UMFPACK library not being compiled with -fPIC that was resolved with this bug fix:

sudo add-apt-repository ppa:bzindovic/suitesparse-bugfix-1319687 

Is this what you are referring to when you say that suitesparse is not available with Ubuntu 14.04?

**Comment by martinandersen** [comment]
I was not aware of that bug fix, but yes, the issue with 14.04 is that SuiteSparse was compiled without  the fPIC flag.

**Comment by martinandersen** [comment]
Can you compile CVXOPT 1.1.9 on 14.04 after applying this bug fix? If yes, then I can mention this in the installation instructions.

**Comment by galbramc** [comment]
Yes it compiles on 14.04 with that bug fix. We use suitesparce with python for another project I'm working on so we already had that fix in our system. I just had to set CVXOPT_SUITESPARSE_INC_DIR=/usr/include/suitesparce.

There is no bug fix though for Ubuntu 15 that I am aware of.

I also tried our 12.04 system, but that version of suitsparse predates the suitesparseconfig library and cvxopt fails on the link command. You don't need that libary if it does not exist. We use CMake for our project and treat the suitesparseconfig library as optional.

I also tried on our 12.04 system. There the umfpack version is older before the suitesparse
